cypress/integration/login_admin.js

```diff
@@ -10,15 +10,15 @@ describe.skip('Admin login test', () => {
 
         // should be visible after Log In click
         cy.get('#login-modal').should('not.be.visible');
-        cy.contains('Log In').click();
+        cy.get('.mobile-hidden').contains('Log In').click();
```

Comment:
It's generally a bad idea to rely on incidental selectors like `.mobile-hidden` as part of cypress testing.

I'm also wondering why this became necessary, did you find that the width of the screen in tests was putting the design in mobile layout and hiding the login button? Did this extra selector make it work even though it was hidden? I wouldn't expect adding this before the same `.contains('Log In')` to fix anything as AFAICT it would just further restrict the contains check?

Reply:
https://github.com/SlateFoundation/slate/commit/98c4de86319284165d10856af3b28e91823469c1#diff-fc3befdd534c43f9355501bd6768ec67a8fcab60563771e0453a06134c35d37a

I _believe_ this PR changed the template so there's two sections, and without specifying `cy.get` is grabbing the first, which happens to be the mobile-only (hidden) Log In.